tools/uninstall.sh

```diff
@@ -20,9 +20,24 @@ then
 
   source ~/.zshrc;
 else
-  echo "Switching back to bash"
+  echo "Changing default login shell from zsh back to bash ... login required."
   chsh -s /bin/bash
-  source /etc/profile
+  if [ "$?" = "0" ]; then
+    echo "Done.  All new shell windows will be bash!"
```

Comment:
Is "window" the right term? The user might not be running a windowing system. Maybe "session"?